python/python/glide/async_commands/core.py

```diff
@@ -5440,6 +5440,41 @@ async def sscan(
             await self._execute_command(RequestType.SScan, args),
         )
 
+    async def watch(self, keys: List[str]) -> TOK:
+        """
+        Marks the given keys to be watched for conditional execution of a transaction. Transactions
+        will only execute commands if the watched keys are not modified before execution of the
+        transaction.
+
```

Comment:
Add a cross slot notice

Reply:
watch is not a cluster command

Reply:
Both clients can execute a transaction and watch the key.

Reply:
Cross-slot means you can provide multiple keys, and it'll fail if the keys map to different slots.  
Just use this: https://github.com/aws/glide-for-redis/blob/d3f3f5fedd4dbfb927ed2d01f4ab1d74273715d6/python/python/glide/async_commands/core.py#L1947-L1948

Reply:
```
When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slots.
```

Reply:
added in latest version